00-RELEASENOTES

```diff
@@ -9,6 +9,69 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.0.0 RC2  -  Released Tue 2 Sep 2024
+================================================================================
+Upgrade urgency LOW: This is the second release candidate of Valkey 8.0, which
+generally includes stability and performance improvements over the first release
+candidate. This release is fully compatible with Redis OSS 7.2.4.
```

Comment:
Shall we say "fully backward compatible"?

Fully compatible sounds a bit like they're equivalent, that Valkey 8.0 is a drop-in replacement. They're not compatible both ways and Valkey 8.0 is clearly a step forward.

Reply:
Valkey 8.0 is a drop in replacement to Redis OSS 7.2 though, it's just better at it :). I would like to continue making it clear it's fully compatible.  I prefer keeping the compatible wording, but open to hear other perspectives.

Reply:
OK